src/main/java/org/cru/godtools/api/v2/DraftResource.java

```diff
@@ -104,6 +106,31 @@ public Response getTranslation(@PathParam("language") String languageCode,
 	}
 
 	@GET
+	@Path("/{language}/{package}/pages")
+	@Produces(MediaType.APPLICATION_JSON)
+	public Response getPages(@PathParam("language") String languageCode,
+							 @PathParam("package") String packageCode,
+							 @HeaderParam("Authorization") String authTokenHeader) throws MalformedURLException
+	{
+		log.info("Requesting draft pages for package: " + packageCode + " for language: " + languageCode);
+
+		AuthorizationRecord.checkAccessToDrafts(authService.getAuthorizationRecord("", authTokenHeader), clock.currentDateTime());
```

Comment:
No real good reason, other than `""` will never create an NPE.  I can change this to `null`